cmd/solana_exporter/slots.go

```diff
@@ -331,6 +334,14 @@ func (c *SlotWatcher) fetchAndEmitSingleFeeReward(
 ) error {
 	block, err := c.client.GetBlock(ctx, rpc.CommitmentConfirmed, slot)
 	if err != nil {
+		var rpcError *rpc.RPCError
+		if errors.As(err, &rpcError) {
+			// this is the error code for slot was skipped:
+			if rpcError.Code == -32007 && strings.Contains(rpcError.Message, "skipped") {
```

Comment:
Can you make that a constant please? Also, where did you get that rpc error code from? The solana source or some docs? If from some docs, can you link them in a comment 🙏

Reply:
No docs, just ran the call on a block that was skipped and looked at the code. Can make it a constant tho

Reply:
[Here you go](https://github.com/anza-xyz/agave/blob/489f483e1d7b30ef114e0123994818b2accfa389/rpc-client-api/src/custom_error.rs#L17) sir!